### ISSUE TITLE
Logs not generated (Windows)

### ISSUE THREAD

**Issue by SimonTurl** [apprenticeharper/DeDRM_tools#748]
Hi,
I wanna try the stand alone solution for Windows instead of the Calibre plugin. I installed python2.7 and pycrypto2.6. Then I followed instructions in the readme file. Finally when I open the .bat shortcut a python instance is open for some second then closed. I can see some error but I have not the time to read. Also since it gets closed I didn't insert any e-ink serial. I have no logs in my user directory. What I did wrong? Thanks in advance

**Comment by ElleKayEm** [comment]
See this from the FAQs:

https://github.com/apprenticeharper/DeDRM_tools/blob/master/FAQs.md#ive-installed-activestate-python-and-pycrypto-but-the-windows-application-wont-run-what-have-i-done-wrong

**Comment by SimonTurl** [comment]
Yes thanks, I already checked also that. I can see the _tcl8.5_ dir inside _lib_ dir and inside the _tcl_ dir (I installed the lastest available activestate python).

**Comment by ElleKayEm** [comment]
Run the batch file from a command window (instead of double-clicking) to see what that error is.  You won't get the log in your user directory until you've attempted to DeDRM a book.

**Comment by SimonTurl** [comment]
Also I already did that. I don't get any error in the windows console since the error is in the python console. The cmd runs successfully the python script and finish with no errors, instead the python script get some error and closes in a sec.

**Comment by ElleKayEm** [comment]
Ah...you might try 6.5.5.  It seems like the last two versions are where people started really having issues with the stand alone apps.  That version won't work with Amazon's KFX format, but you're better off using the calibre plugin for those anyway.

**Comment by SimonTurl** [comment]
After an hour of attempts.. here is a partial log, hope it helps to understand the problem. And yes thanks, the workaround worked! Version 6.5.5 is still working here.

![error](https://user-images.githubusercontent.com/46942740/51765798-1b541c80-20d9-11e9-9827-0e5ea4951f9e.png)

**Comment by ElleKayEm** [comment]
That looks familiar....see Issue #512 and others that have been referenced there.
